### ISSUE TITLE
Update website to remove Sysdig language

### ISSUE THREAD

**Issue by kris-nova** [falcosecurity/falco#903]
As it stands there is some misleading documentation on [falco.org](https://github.com/cncf/foundation/blob/master/website-guidelines.md)

Particularly where we mention language such as:

> Powered by Sysdig’s system call capture infrastructure

Can we please update the website to follow the [official website guidelines](https://github.com/cncf/foundation/blob/master/website-guidelines.md) that the CNCF suggests. 

CC @caniszczyk @dankohn

**Comment by leodido** [comment]
Totally agree +1

**Comment by mfdii** [comment]
This is left there because we are indeed powered by Sysdig’s capture technology that is available via the dependencies we pull in via sysdig OSS. If we’re looking for clarity that we aren’t referring to sysdig as a company I’d suggest we make it read “sysdig OSS” and link to the OSS repo

**Comment by kris-nova** [comment]
Yes - I mentioned this when it was brought up. We just need to be clear that we have a very clean difference between

1) Sysdig the company
2) Falco uses `libscap` and `libsinsp` c++ libraries used in tools the the `sysdig` CLI tool.

**Comment by bencer** [comment]
On the home page this is used in a way that follows the guidelines:

"Leveraging Sysdig’s open source Linux kernel instrumentation, Falco gains deep insight into system behavior"

**Comment by fntlnz** [comment]
This issue should be in `falcosecurity/falco-website`. Let's transfer?

**Comment by leodido** [comment]
Let's transfer it to the right repo, yes
